src/components/checkbox/checkbox.tsx

```diff
@@ -5,39 +5,55 @@ import { Svg, Path, Rect } from 'react-native-svg';
 import { EventFn } from '../../config/general';
 import { formatSize } from '../../styles/format-size';
 import { generateHitSlop } from '../../styles/generate-hit-slop';
+import { useColors } from '../../styles/use-colors';
 import { CheckboxStyles } from './checkbox.styles';
 
 interface Props {
+  disabled?: boolean;
   value: boolean;
   size?: number;
   strokeWidth?: number;
   onChange: EventFn<boolean>;
 }
 
 export const Checkbox: FC<Props> = ({
+  disabled,
   value,
   size = formatSize(24),
   strokeWidth = formatSize(1.5),
   children,
   onChange
```

Comment:
Try to keep the order of props the same overywhere
<img width="634" alt="Screenshot 2021-07-16 at 16 28 48" src="https://user-images.githubusercontent.com/10281464/125955701-66cde716-433b-4c49-8291-903805f3932a.png">
![image](https://user-images.githubusercontent.com/10281464/125955726-c4edb1fc-be4c-4c0e-9220-00f00d444828.png)